Tests/scripts/run_installer_on_instance.sh

```diff
@@ -22,6 +22,9 @@ USER="centos"
 echo "wait 90 seconds to ensure server is ready for ssh"
 sleep 90s
 
+echo "add instance to known hosts"
+ssh-keyscan -H ${PUBLIC_IP} >> ~/.ssh/known_hosts
```

Comment:
can you explain why this step is needed?

Reply:
adding the public ip to the known host to allow the docker to ssh to the machine (otherwise access is declined)